src/braket/circuits/circuit.py

```diff
@@ -484,6 +530,9 @@ def add_circuit(
             values = target
             target_mapping = dict(zip(keys, values))
 
+        if circuit.has_free_parameters:
+            self._parameterized = True
```

Comment:
Wouldn't the "add_instruction" call already do this? Why do we need to do it twice?

Reply:
Removed parameterized from circuit